cmd/k8s-bigip-ctlr/main_test.go

```diff
@@ -234,9 +235,13 @@ var _ = Describe("Main Tests", func() {
 		Expect(err).To(BeNil())
 		Expect(proc).ToNot(BeNil(), "Should have process object.")
 
-		cmd = exec.Command("bash", []string{"test/testPyTest.sh"}...)
+		cmd := exec.Command("bash", []string{"test/testBigipconfigdriver.sh"}...)
 		session, _ := Start(cmd, GinkgoWriter, GinkgoWriter)
 		Eventually(session, 30*time.Second).Should(Exit(0))
+
```

Comment:
Move this test into its own context, then call a BeforeEach to set up and start the driver. That way you can place the killDriver block into an AfterEach to guarantee the subprocess is deleted even on test failures.

Reply:
Updated the PR with changes requested. There are more Indentation changes becase I added a hierarchy to the tests.